kiwixbuild/dependencies/base.py

```diff
@@ -527,7 +527,7 @@ def library_type(self):
         return 'static' if self.buildEnv.platformInfo.static else 'shared'
 
     def _configure(self, context):
-        context.no_skip = False
+        context.no_skip = True
```

Comment:
You should remove the line. It it set to `True` or `False` automatically depending if this is one of our project or not.

Reply:
Yep I figure that after it failed. I thought only our projects were using meson for some reason